src/jwplayer/players/shared/JwPlayerWrapperWithStrategyRules.tsx

```diff
@@ -71,6 +74,16 @@ const JwPlayerWrapperWithStrategyRules: React.FC<JwPlayerWrapperProps> = ({
 			addBaseTrackingEvents(playerInstance);
 		});
 
+		playerInstance.on(JWEvents.AD_IMPRESSION, () => {
+			const newAdIndex = adIndex + 1;
+			const newPrerollAdTag = updateRVParam(prerollAdTag, newAdIndex);
```

Comment:
Analogous request as the one in the previous comment. Terms "tag" and "url" represent different concepts.